autoload/slime.vim

```diff
@@ -250,21 +250,18 @@ function! s:DtachConfig() abort
 endfunction
 
 """""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
-" Tabbed
+" Custom
 """""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
 
-function! s:TabbedSend(config, text)
+function! s:CustomSend(config, text)
   call system("xdotool type --delay 0 --window " . b:slime_config["window_id"] . " -- " . shellescape(a:text))
 endfunction
 
-function! s:TabbedConfig() abort
+function! s:CustomConfig() abort
   if !exists("b:slime_config")
     let b:slime_config = {"window_id": ""}
   endif
-  if !exists("b:slime_custom_script")
-    let b:slime_custom_script = tabbed
-  endif
-  let b:slime_config["window_id"] = trim(system("b:slime_custom_script"))
+  let b:slime_config["window_id"] = trim(system("tabbed-slime"))
```

Comment:
I am not sure how to add a global variable in vim, and utilize it in `trim(system())`. But I export those long code into the `tabbed-slime` script, and do some modification. The content of `tabbed-slime` is:

```
xwininfo -children -id $XEMBED | grep '     0x' | awk '{print $1, $2}' | tail -n +2 | dmenu -i -l 5 -p 'REPL window: ' | cut -b 1-9 | xargs printf '%d'
```

Could I ask how to assign a global variable and put it in this chunk of vim script?

Thank you!

Reply:
`let g:var_name = 5` sets a global var

same for using it: `let b:slime_config["window_id"] = trim(system(g:var_name))`